### PR TITLE
Support lasso flag switching via URL;

### PR DESCRIPTION
**Pull request by seangates**
## Description
Updates the demo site to support adding the design system to the URL. Uses a Lasso flag.

## Context
We need to be able to test the components with DS4, and this will provide the necessary switch.

**Examples:**
```
http://localhost:3000/ds4/ebay-combobox/
http://localhost:3000/ds6/ebay-combobox/
```

### CODE REVIEW

File: demo/index.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ app.get('/', (req, res) => {
     res.redirect(301, '/ebay-button');
```
[the remark this answers is not in the document]
Comment (seangates) [reply demo/index.js]: Yeah, this was quick. I can also handle for those.

Really, this is for us, the core developers, and most folks won't ever run the component library this way. But I can make those changes to make sure we have the functionality we need.

**Comment by DylanPiercey** [comment]
👍 to @yomed's requested changes. Otherwise LGTM.

**Comment by ianmcburnie** [comment]
Can we get this in now? It's beginning to hold up the other PRs.

**Comment by seangates** [comment]
@ianmcburnie Working on it as we speak.

**Comment by seangates** [comment]
@yomed Sorry I missed the var naming. Merged!

**Pull request merged**